pkg/internal/controller/controller_test.go

```diff
@@ -164,7 +163,7 @@ var _ = Describe("controller", func() {
 			c = &cacheWithIndefinitelyBlockingGetInformer{c}
 			ctrl.startWatches = []watchDescription{{
 				src: &singnallingSourceWrapper{
-					SyncingSource: source.Kind(c, &appsv1.Deployment{}),
+					SyncingSource: source.Kind[client.Object](c, &appsv1.Deployment{}, &handler.EnqueueRequestForObject{}),
```

Comment:
I assume we don't use a TypedEnqueueRequestForObject to show that we don't have to?

Reply:
No, just because this was the smaller change :P